include/crud-posts.php

```diff
@@ -197,10 +197,12 @@ public function set_object_terms( $object_id, $terms, $tt_ids, $taxonomy ) {
 	 */
 	public function wp_insert_post_parent( $post_parent, $post_id ) {
 		$lang = $this->model->post->get_language( $post_id );
+		$parent_post_type = $post_parent > 0 ? get_post_type( $post_parent ) : null;
```

Comment:
This also prevents to run `get_translation()` when the post type is translated:

```suggestion
		if ( $parent_id <= 0 ) {
			return $parent_id;
		}

		$lang = $this->model->post->get_language( $post_id );
		$parent_post_type = get_post_type( $post_parent );
```

Reply:
You mean `post_parent` in the new condition you suggest?

Reply:
Forget it, if `$post_parent` is equal to `0`, then `$parent_post_type` will be `false`, then `get_translation()` won't run.